dist/hello.all.js

```diff
@@ -1718,17 +1721,17 @@ hello.utils.Event.call(hello);
 			}
 
 			// Access_token has been removed
-			else if (!session.access_token && oldSess.access_token) {
+			else if (hello.settings.autoemit && !session.access_token && oldSess.access_token) {
 				emit('logout');
 			}
 
 			// Access_token has been created
-			else if (session.access_token && !oldSess.access_token) {
+			else if (hello.settings.autoemit && session.access_token && !oldSess.access_token) {
```

Comment:
The point of this PR is to avoid these events from being emitted, just because there happens to be a `hello` in localstorage. Some people may well want this, but I do not, and I would like to be able to just disable it during init.